src/server/graphql/models/utils.js

```diff
@@ -6,14 +6,16 @@ export const defaultResolveFn = (source, args, {fieldName}) => {
 };
 
 export function resolveForAdmin(source, args, ref) {
-  return ref.rootValue && ref.rootValue.authToken && ref.rootValue.authToken.isAdmin ? defaultResolveFn.apply(this, arguments) : null;
+  return ref.rootValue &&
+    ref.rootValue.authToken &&
+    ref.rootValue.authToken.isAdmin ? defaultResolveFn.apply(this, [source, args, ref]) : null;
 }
 
-export const errorObj = obj => {
```

Comment:
Since this wraps multiple lines with the comments, I think it would be better to use:

```javascript
... (obj) => {
  // comment 1
  // comment 2
  return new Error(...);
}
```